access/slack/app.go

```diff
@@ -2,28 +2,27 @@ package main
 
 import (
 	"context"
-	"net/url"
 	"time"
 
 	"github.com/gravitational/teleport-plugins/access"
 	"github.com/gravitational/teleport-plugins/lib"
 	"github.com/gravitational/teleport-plugins/lib/logger"
+	"github.com/gravitational/teleport/api/types"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/backoff"
 
 	"github.com/gravitational/trace"
 )
 
 // MinServerVersion is the minimal teleport version the plugin supports.
-const MinServerVersion = "4.3.0"
+const MinServerVersion = "6.0.0-alpha.2"
```

Comment:
Are we deliberately keeping compatibility with `6.0` or is this just what was being used to test against?  Seems to me that these changes make more sense if they come alongside `6.1` since that is when we're adding approval/denail to the Web UI.

Reply:
We are targeting 6.1 version, so compatibility with 6.0 is not required. Our plugins will follow the same versioning scheme as Teleport, so the next plugins version will be 6.1

Reply:
@fspmarshall @klizhentas 

It's temporarily! Teleport version in this branch is still `6.0.0-alpha.2` and until Teleport 6.1 is released, the plugin will refuse to start.